libs/s25main/Loader.cpp

```diff
@@ -613,14 +614,59 @@ void Loader::fillCaches()
             }
         }
 
-        glSmartBitmap& bmp = boundary_stone_cache[nation];
+        {
+            glSmartBitmap& bmp = boundary_stone_cache[nation];
+            bmp.reset();
 
-        bmp.reset();
+            bmp.add(GetNationPlayerImage(nation, 0));
+            bmp.addShadow(GetNationImage(nation, 1));
+
+            stp->add(bmp);
+        }
 
-        bmp.add(GetNationPlayerImage(nation, 0));
-        bmp.addShadow(GetNationImage(nation, 1));
+        libsiedler2::Archiv& romBobs = GetArchive("rom_bobs");
+        const auto getAlternative = [&romBobs](unsigned id, unsigned altId) {
+            auto* bmp = convertChecked<glArchivItem_Bitmap_Player*>(romBobs[id]);
+            return bmp ? bmp : convertChecked<glArchivItem_Bitmap_Player*>(romBobs[altId]);
+        };
+        // Special handling for non-native nats: Use roman animations if own are missing
+        const unsigned fallbackNation = nation < NUM_NATIVE_NATS ? nation : NAT_ROMANS;
```

Comment:
can we perhaps rename NATS to NATIONS? `NUM_NATIVE_NATIONS` would make it more readable

Reply:
Good point. Done for this and `NUM_NATS`